fr/how-to/create-site-to-site-vpn-on-vpc.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Créer un VPN site-à-site pour un VPC"
+slug: create-site-to-site-vpn-on-vpc
+---
+
+
+Un VPN site-à-site est utile pour interconnecter un VPC à un bureau à distance, à un autre centre de données, ou à un autre VPC.
+
+L'exemple suivant illustre comment à connecter deux VPCs avec un site-à-site VPN.  Les détails varient selon vos appareils.
+
+#### Détails pour les VPCs d'exemple
+| Nom du VPC | Nom de l'environnement | CIDR du VPC | IP NAT source |
+| --- | --- | --- | --- |
+| dmz | env-dmz | 10.0.0.0/22 | 172.30.200.106 |
+| gestion | env-gest | 10.0.4.0/22 | 172.30.200.107 |
+
+#### Configuration du VPN site-à-site
+1. Aller à l'environnement **dmz**.
+1. Cliquer sur l'onglet *Réseautique*.
+1. Cliquer sur l'icône d'engrenage pour *VPNs site-à-site*.
+1. Cliquer sur *Ajouter VPN site-à-site*.
```

Comment:
I would either add 'en haut à droite de la page' or remove it from the english

Reply:
Good catch, deleted it from the English.